EventVisualisation/Base/src/EventManager.cxx

```diff
@@ -9,54 +9,54 @@
 // or submit itself to any jurisdiction.
 
 ///
-/// \file    EveInitializer.cxx
+/// \file    EventManager.cxx
 /// \author  Jeremi Niedziela
 
-#include "EventManager.h"
+#include "EventVisualisationBase/EventManager.h"
 
-#include "MultiView.h"
+#include "EventVisualisationDataConverter/MinimalisticEvent.h"
+#include "EventVisualisationBase/Track.h"
+#include "EventVisualisationBase/ConfigurationManager.h"
 
 #include <TEveManager.h>
 #include <TEveProjectionManager.h>
 #include <TSystem.h>
+#include <TEnv.h>
+#include <TEveElement.h>
+#include <TEveTrackPropagator.h>
+#include <TGListTree.h>
+#include <TEveTrack.h>
 
 using namespace std;
 
 namespace o2  {
 namespace EventVisualisation {
 
-EventManager* EventManager::sMaster  = nullptr;
+EventManager* EventManager::sInstance  = nullptr;
 
 EventManager* EventManager::getInstance()
 {
-  if(!sMaster){
-    new EventManager();
-  }
-  return sMaster;
+  if(!sInstance){new EventManager();}
+  return sInstance;
 }
 
-void EventManager::registerEvent(TEveElement* event)
+string EventManager::getDataTypeName(EDataType type)
```

Comment:
`const char*` as return type?

Reply:
This will be used for some string manipulations, which is much easier with std::string than with const char*. Is there a strong motivation for replacing std::string with const char*?